tests/test_fix_data_type.py

```diff
@@ -0,0 +1,62 @@
+import unittest
```

Comment:
I think something went wrong with this whole file, because now all of the tests you had previously have been deleted. I would recommend reverting to the version of this file from the previous commit and then turning that into a unit test to align with the other test files.